app/components/cloud-call.js

```diff
@@ -110,7 +94,6 @@ export default Component.extend({
       if (!shouldRender) return;
```

Comment:
disabling the render loop requires restarting it from the handleScroll function.  Don't know the ember-y way to do this.
I guess you would need the break all the functions out of ```didInsertElement```... then didInsertElement would init everything and do ```this.brokenOutRender.bind(this)```?  idk seems like a mess